content/en/docs/instrumentation/java/automatic/_index.md

```diff
@@ -83,6 +83,6 @@ might want to [annotate](annotations) selected methods or add [manual
 instrumentation](../manual) to collect custom telemetry data.
 
 [Agent Configuration]: agent-config
-[opentelemetry-javaagent.jar]: https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v1.12.0/opentelemetry-javaagent.jar
+[opentelemetry-javaagent.jar]: https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v1.17.0/opentelemetry-javaagent.jar
```

Comment:
For the [getting started](https://opentelemetry.io/docs/instrumentation/java/getting-started/) we move to the latest link github provides, would that work as well?

`https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/latest/download/opentelemetry-javaagent.jar`

Reply:
Good idea, will do.